planetiler-core/src/main/java/com/onthegomap/planetiler/reader/WithTags.java

```diff
@@ -1,8 +1,9 @@
 package com.onthegomap.planetiler.reader;
 
+import java.util.Map;
```

Comment:
I'm not sure why this moved, it's what the formatter did on its own 🤔

Reply:
as long as `spotless:check` is happy then 🤷